hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RegionServerAccounting.java

```diff
@@ -67,6 +70,7 @@ public RegionServerAccounting(Configuration conf) {
     this.globalOnHeapMemstoreLimit = MemorySizeUtil.getOnheapGlobalMemStoreSize(conf);
     this.globalOnHeapMemstoreLimitLowMark =
         (long) (this.globalOnHeapMemstoreLimit * this.globalMemStoreLimitLowMarkPercent);
+    this.retainedRegionRWRequestsCnt = new ConcurrentHashMap<>();
```

Comment:
This data structure can grow w/o bound?

This data structure needs a comment on how it is used and life cycle of items. They items should age out I'd think.

Reply:
> This data structure can grow w/o bound?
The entry of it will be removed in time, so will not grow too big.

> This data structure needs a comment on how it is used and life cycle of items. They items should age out I'd think.
Added some comment.

Reply:
So, this data structure will get an entry when we close a region that we are to reopen on this server? On open, the entry is removed?

s/reigon/region/

Reply:
Does the reopen of regions on same server happen often?

Reply:
> So, this data structure will get an entry when we close a region that we are to reopen on this server? On open, the entry is removed?
> 
> s/reigon/region/

Yeah.

Reply:
> Does the reopen of regions on same server happen often?

It will happen when alter table.